### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ghost**
updating for consistency with template

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,18 +46,18 @@ Developers can include the Java AMQP 0-9-1 Client using the following maven depe
 
 # Running the WebSocket demo
 
-## From the command-line
+## Run From the Command-Line
 0. Change directory: ```cd ws/demo/target```
```
Comment (vjwang) [inline README.md:50]: Remove the hyphen: Command Line.

File: README.md [hunk 2]
```diff
@@ -46,18 +46,18 @@ Developers can include the Java AMQP 0-9-1 Client using the following maven depe
...
 0. Import the project in Eclipse
 0. Under `gateway.client.java.demo` project, right-click on `WebSocketFrame.java` or `WebSocketApplet.java` in
-src/main/java/org.kaazing.net.ws.demo and run!
+`src/main/java/org.kaazing.net.ws.demo` and run.
 
 # Running the AMQP 0-9-1 demo
 
-## From the command-line
+## From the Command-Line
 0. Change directory: ```cd amqp0-9-1/demo/target```
```
Comment (vjwang) [inline README.md:61]: No need for a hyphen here, either: Command Line

File: README.md (hunk 1 above)
Comment (ghost) [inline README.md:50]: https://en.wikipedia.org/wiki/Command-line_interface. Unless you have a Chicago Style Guide link for me? :)

File: README.md (hunk 2 above)
Comment (ghost) [inline README.md:61]: see above

File: README.md (hunk 1 above)
Comment (vjwang) [inline README.md:50]: The hyphen is there as a description of "interface." The wikipedia entry you linked refers to "command lines" (no hyphen, without the "interface" object).

Comment (ghost) [inline README.md:50]: it is an interface.
I'll change the heading to "Command-Line Interface (CLI)"

Comment (vjwang) [inline README.md:50]: SGTM

**Pull request merged**